msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -20,6 +20,16 @@ Your bot receives messages from Teams using the `Text` property and it sends sin
 
 For more information, see [user attribution for bot messages](/microsoftteams/platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit?tabs=dotnet%2Cdotnet-1#user-attribution-for-bots-messages).
 
+The following table lists the events that your bot can receive and take action on:
```

Comment:
Could you change all "event" to "activity" for message edit event, message soft delete event, and message undelete event? So that it will be consistent with the message activity. 

Also, could you update the title "Receive a message" to "Recieve a message activity"?

Reply:
Updated as suggested